runtime/vm/CRIUHelpers.cpp

```diff
@@ -1836,6 +1836,22 @@ criuCheckpointJVMImpl(JNIEnv *env,
 
 		VM_VMHelpers::setVMState(currentThread, J9VMSTATE_CRIU_SUPPORT_CHECKPOINT_PHASE_END);
 
+#if defined(LINUX)
+		if (J9_ARE_ALL_BITS_SET(vm->extendedRuntimeFlags2, J9_EXTENDED_RUNTIME2_CRIU_DISCLAIM_CLASSES)) {
+			systemReturnCode = vm->internalVMFunctions->disclaimAllClassMemory(vm);
```

Comment:
do an experiment where you call disclaim twice just to make sure it works as intended

Reply:
It seems like the experiment works as intended.